### PR TITLE
Even nicer page

### PR DESCRIPTION
**Pull request by Morta-Ant**
ugly page -> pretty page

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ name = "pypi"
 [packages]
 flask = "*"
 flask-nav = "*"
+fancybox = "*"
```
Comment (eustaceb) [inline Pipfile]: Fancybox for Python is a different thing - it's used for printing message boxes on the commandline, see https://pypi.org/project/fancybox/
```suggestion

```

File: hello.py [hunk 2]
```diff
@@ -2,16 +2,17 @@
 from flask_nav import Nav
 from flask_nav.elements import Navbar, View, Subgroup
 import json
+import fancybox
```
Comment (eustaceb) [inline hello.py]:
```suggestion

```

**Review by eustaceb — CHANGES_REQUESTED** [review]
Looks good overall, just need to remove the `fancybox` python dependency.

File: Pipfile (hunk 1 above)
Comment (Morta-Ant) [reply Pipfile]: fuk off fancybox why u so confuse

File: hello.py (hunk 2 above)
Comment (Morta-Ant) [reply hello.py]: fannybox :(

**Review by eustaceb — APPROVED** [review]
Looks good

**Pull request merged**